liesel/model/nodes.py

```diff
@@ -1050,6 +1115,79 @@ class Var:
     >>> x
     Var(name="x")
 
+    .. rubric:: Accessing inputs
+
+    :class:`.Calc` and :class:`.Dist` objects support access to their inputs via
+    square-bracket syntax. Thus, with a :class:`.Var` object, you can use square bracket
+    indexing on its attributes :attr:`.Var.value_node` and :attr:`.Var.dist_node`.
+    You can access both named and unnamed arguments this way.
```

Comment:
I think positional arguments is a better/more commonly used term here. Use instead of unnamed.

Reply:
Thanks, I implemented it :)